Change the Name showed by /proc/cpuinfo



#
# Patch managed by http://www.holgerschurig.de/patcher.html
#

--- linux-2.4.17_mvl21/arch/arm/mach-sa1100/assabet.c~machine_name
+++ linux-2.4.17_mvl21/arch/arm/mach-sa1100/assabet.c
@@ -374,7 +374,7 @@
 }
 
 
-MACHINE_START(ASSABET, "Intel-Assabet")
+MACHINE_START(ASSABET, "Tradesquare.NL Tuxpad 1")
 	BOOT_MEM(0xc0000000, 0x80000000, 0xf8000000)
 	BOOT_PARAMS(0xc0000100)
 	FIXUP(fixup_assabet)
